examples/navier-stokes/advection.h

```diff
@@ -60,7 +60,29 @@ struct AdvectionContext_ {
 #endif
 
 // *****************************************************************************
-// TODO: Annotate this function
+// This QFunction sets the the initial conditions and boundary conditions
+//
+// Initial Conditions:
+//   Mass Density:
+//     Constant mass density of 1.0
+//   Momentum Density:
+//     Rotational field in x,y with no momentum in z
+//   Energy Density:
+//     Maximum of 1. x0 decreasing linearly to 0. as radial distance increases
+//       to (1.-r/rc), then 0. everywhere else
+//
+//  Boundary Conditions:
+//    Mass Density:
+//      0.0 flux
+//    Momentum Density:
+//      0.0
+//    Energy Density:
+//      0.0 flux
+//
+// *****************************************************************************
+
+// *****************************************************************************
+// This helper function provides the current 3D advection IC formulation
```

Comment:
current?

Reply:
Yeah. I'm guessing since I didn't write the code, but it seems like a big purpose of these helper functions is the ability to more easily switch out ICs with less boilerplate.

Reply:
If there is a more natural description, anyone is welcome to replace it